src/CommunityVoices/App/Website/View/Image.php

```diff
@@ -225,10 +225,6 @@ public function getAllImage($request)
         return $response;
     }
 
-    public function getImageUpload($request)
```

Comment:
Just be sure to do this on the API side too.

Reply:
Maybe you did this?  The last of my open comments are about it.

Reply:
done